core/drivers/tzc380.c

```diff
@@ -228,7 +228,17 @@ int tzc_auto_configure(vaddr_t addr, vaddr_t size, uint32_t attr,
 	uint64_t lsize = size;
 	uint32_t mask = 0;
 	int i = 0;
-	uint8_t pow = TZC380_POW;
```

Comment:
This define can be removed as well in the header file.

Reply:
Right! Fixed